pimd/pim_iface.c

```diff
@@ -1504,6 +1504,29 @@ void pim_if_create_pimreg(struct pim_instance *pim)
 	}
 }
 
+struct prefix* pim_if_find_connected_source(struct interface *ifp, struct in_addr src)
+{
+	struct listnode *cnode;
+	struct connected *c;
+	struct prefix p;
+
+	if (!ifp)
+		return 0;
```

Comment:
Please only ever use NULL when you mean it.  The c standard does not guarantee that 0 is NULL ( although by convention it typically is ).